pkg/specgen/generate/oci.go

```diff
@@ -183,32 +184,32 @@ func (s *SpecGenerator) ToOCISpec(rt *libpod.Runtime, newImage *image.Image) (*s
 
 	// NAMESPACES
 
-	if err := s.pidConfigureGenerator(&g); err != nil {
+	if err := pidConfigureGenerator(&g, s); err != nil {
```

Comment:
```suggestion
	if err := pidConfigureGenerator(s, &g); err != nil {
```
A small nit since s is the receiver, I would like to see it first not last.